src/convert.cpp

```diff
@@ -38,6 +42,11 @@ void setNumberDataPerLine(const unsigned int nd) { nbData = nd; }
 
 void setShowDataContent(const bool dc) { showDataContent = dc; }
 
+void setOutputBitLength(const unsigned int bl) { if ((bl==16)||(bl==32)) outlen = bl;}
```

Comment:
Was this meant to support 64-bit as well? Help string says "Generate header with 16 32 or 64 bit int types"

Reply:
Probably I was too fast. For now I will remove 64 bits. 16 and 32bits are usefull for Audio files